index.html

```diff
@@ -1,17 +1,63 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>My Portfolio</title>
-    <link href="style.css">
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <nav>
+      <a href="index.html">DAPHINE</a>
+      <div class="humburger-btn">
+        <span class="span-1"></span>
+        <span class="span-2"></span>
+        <span class="span-3"></span>
+      </div>
```

Comment:
- [ ] Let's ensure that this is placed within a `<nav>` tag because it is part of the navigation. Later on, when you click on this hamburger button, it will display the mobile navigation.
- [ ] As mentioned before you will be using hamburger as a `button` later on so let's make sure we take our preparation and implement it as a button for now